src/ui/pages/History/ActionsList/ActionsList.tsx

```diff
@@ -55,7 +55,7 @@ export function ActionsList({
                 const hash = addressAction.transaction.hash;
                 return {
                   key: isLocalAddressAction(addressAction)
-                    ? addressAction.relatedTransaction || hash
+                    ? `local-${addressAction.relatedTransaction || hash}`
```

Comment:
why do we need this?

Reply:
It's not related but I decided not to create a separate PR for this
It fixed react error that multiple components for same key are rendered. Turns out it happens when you try to accelerate a tx and initial tx gets mined